MakeWish/interconnweb
# #168950062  register user

### What the PR does?
- creates factories
- created models
- creates seeders and makes migrations
- provides a route for the user to use in registration

### How to test it?
- pull the branch
- set the environment for the database by populating the .env file just like the `example.env file` ie

- [x] db_connection=pqsql

- [x] db_user=postgres
- source the `.env` file in the terminal
- Ensure you have node setup locally and run `npm install` for the Vue installed when running make auth, that supports PHP blades.
- run `composer install` and run `php artisan serve`
- control-click on the link in the terminal and go to the browser.
- Click register and all is well


### storyId
[#168950062](https://www.pivotaltracker.com/story/show/168950062)


![upload](https://user-images.githubusercontent.com/43015966/67818585-d2475380-fac2-11e9-8ade-3b1443e55c16.JPG)
